server/api/core/core.js

```diff
@@ -301,15 +314,27 @@ export default {
         emails: {
           address: options.email,
           verified: true
-        },
-        domains: Meteor.settings.ROOT_URL
+        }
       }
     });
 
+    // get the current shop
+    const shop = Shops.findOne(shopId);
+
+    // add the current domain to the shop if it doesn't already exist
+    if (!_.includes(shop.domains, this.getDomain())) {
```

Comment:
Our goal should be to use native methods as much as possible, someday maybe no underscore or lodash.. `shop.domains.includes(this.getDomain());`?